packages/legend-extension-dsl-data-space/src/components/query/DSL_DataSpace_LegendQueryApplicationPlugin.tsx

```diff
@@ -156,6 +158,7 @@ export class DSL_DataSpace_LegendQueryApplicationPlugin extends LegendQueryAppli
                   _query.stereotypes = query.stereotypes;
                   _query.content = query.content;
                   _query.owner = query.owner;
+                  _query.lastUpdatedt = query.lastUpdatedt;
```

Comment:
Is the `lastUpdatedt` intentional? Maybe camelcase or lastUpdated?

Reply:
thanks for pointing this. it's a typo. It's supposed to be `lastUpdatedAt`